apps/docs/src/guide/types/address.md

```diff
@@ -1,23 +1,53 @@
 # Address
 
-`Address` is a wrapper on `Bytes32` and `Bech32`. The available interface of the TS-SDK Address contains many useful functions to switch between Address formats:
+In Sway, the `Address` type serves as a type-safe wrapper around the primitive `b256` type. The SDK takes a different approach and has its own abstraction for the `Address` type.
 
-<<< @/../../../packages/interfaces/src/index.ts#AbstractAddress{ts:line-numbers}
+## AbstractAddress Class
 
-These are the main ways of creating an `Address`
+The SDK defines the `AbstractAddress` class, which provides a set of utility functions for easy manipulation and conversion between address formats.
 
-From a [Bech32](./bech32.md) Address:
+<<< @/../../../packages/interfaces/src/index.ts#address-1{ts:line-numbers}
 
-<<< @/../../../packages/fuel-gauge/src/doc-examples.test.ts#Address-bech32{ts:line-numbers}
+## Address Class
 
-From a Public Key:
+Aside conforming with the interface from `AbstractClass` class, the `Address` class defines one property: `bech32Address`, wich is of the [Bech32](./bech32.md) type.
 
-<<< @/../../../packages/fuel-gauge/src/doc-examples.test.ts#Address-publicKey{ts:line-numbers}
+<<< @/../../../packages/address/src/address.ts#address-2{ts:line-numbers}
 
-From a [256 Bit Address](./bits256.md) Address:
+## Creating an Address
 
-<<< @/../../../packages/fuel-gauge/src/doc-examples.test.ts#Address-b256{ts:line-numbers}
+Thanks to the utility functions provided by the `AbstractAddress` class, there are several ways to create an `Address` instance:
 
-You can also use a number of utilities as part of the Address Class:
+### From a Bech32 Address
 
-<<< @/../../../packages/fuel-gauge/src/doc-examples.test.ts#Address-utils{ts:line-numbers}
+To create an `Address` from a `Bech32` address, use the following code snippet:
+
+<<< @/../../docs-snippets/src/guide/types/address.test.ts#address-2{ts:line-numbers}
+
+### From a Public Key
+
+To create an `Address` from a public key, use the following code snippet:
+
+<<< @/../../docs-snippets/src/guide/types/address.test.ts#address-3{ts:line-numbers}
+
+### From a 256-bit Address
+
+To create an `Address` from a 256-bit address, use the following code snippet:
+
+<<< @/../../docs-snippets/src/guide/types/address.test.ts#address-4{ts:line-numbers}
+
+## Utility Functions
+
+The `Address` class also provides some pratical utility functions:
+
+1. Create a new `Address` from an ambiguous source that may be a Bech32 or B256 address:
```

Comment:
This list could be structured better like this:

```suggestion
1. `fromString`: Create a new `Address` from an ambiguous source that may be a Bech32 or B256 address:
```

i.e. let's add the name of the helper before the description in each point.